db/seeds.rb

```diff
@@ -5,3 +5,24 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+require 'rubygems'
+require 'nokogiri'
+require 'open-uri'
+
+page = Nokogiri::HTML(open("http://www.en365.ru/top1000b.htm"))
+rows = page.css('td#middle tr')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.